app/bundles/LeadBundle/Controller/AjaxController.php

```diff
@@ -780,6 +786,10 @@ protected function updateLeadFieldValuesAction(Request $request)
                     0 => $properties['no'],
                     1 => $properties['yes'],
                 ];
+            } elseif ($leadFieldType == 'boolean') {
+                $fieldHelper = new FormFieldHelper();
+                $fieldHelper->setTranslator($this->factory->getTranslator());
```

Comment:
Please use `$this->get('translator')` instead of `$this->factory->getTranslator()`